2013/11/13 10:38:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2013/11/13 10:39:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2013/11/13 10:39:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2013/11/13 10:39:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2013/11/13 10:39:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii
---
2013/11/13 10:40:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:41:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:41:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:41:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:41:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:41:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:41:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:34 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:42:38 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:45:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:45:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:45:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:45:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:46:49 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:46:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:51:27 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:51:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:54:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: YES)' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://www.oxbridge.com/index.php?r=gii/default/index
---
2013/11/13 10:59:37 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'oxbridge.user' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `user`.
in /home/lunjiang/workspace/oxbridge/oxbridge/index.php (13)
2013/11/13 11:01:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2013/11/13 11:01:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "User".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('User')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=User
---
2013/11/13 11:01:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user/list".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user/list')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user/list
---
2013/11/13 11:01:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "User".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('User')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=User
---
2013/11/13 11:01:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=user
---
2013/11/13 11:02:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gii/User".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/User')
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=gii/User
---
2013/11/13 12:29:29 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "setUserType".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CComponent.php:265
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(225): CComponent->__call('setUserType', Array)
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): CActiveRecord->__call('setUserType', Array)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): User->setUserType('type', Array)
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(User), 'type')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModel.php(159): CValidator->validate(Object(User), NULL)
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(794): CModel->validate(NULL)
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php(78): CActiveRecord->save()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#18 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://www.oxbridge.com/index.php?r=user/create
---
2013/11/13 12:30:02 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "setUserType".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CComponent.php:265
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(225): CComponent->__call('setUserType', Array)
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): CActiveRecord->__call('setUserType', Array)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): User->setUserType('type', Array)
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(User), 'type')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModel.php(159): CValidator->validate(Object(User), NULL)
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(794): CModel->validate(NULL)
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php(78): CActiveRecord->save()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#18 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://www.oxbridge.com/index.php?r=user/create
---
2013/11/13 12:30:04 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "setUserType".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CComponent.php:265
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(225): CComponent->__call('setUserType', Array)
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): CActiveRecord->__call('setUserType', Array)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): User->setUserType('type', Array)
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(User), 'type')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModel.php(159): CValidator->validate(Object(User), NULL)
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(794): CModel->validate(NULL)
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php(78): CActiveRecord->save()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#18 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://www.oxbridge.com/index.php?r=user/create
---
2013/11/13 12:30:27 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "setUserType".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CComponent.php:265
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(225): CComponent->__call('setUserType', Array)
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): CActiveRecord->__call('setUserType', Array)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): User->setUserType('type', Array)
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(User), 'type')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModel.php(159): CValidator->validate(Object(User), NULL)
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(794): CModel->validate(NULL)
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php(78): CActiveRecord->save()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#18 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://www.oxbridge.com/index.php?r=user/create
---
2013/11/13 12:30:33 [warning] [application] Failed to set unsafe attribute "password_repeat" of "User".
in /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php (77)
in /home/lunjiang/workspace/oxbridge/oxbridge/index.php (13)
2013/11/13 12:30:33 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "setUserType".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CComponent.php:265
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(225): CComponent->__call('setUserType', Array)
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): CActiveRecord->__call('setUserType', Array)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): User->setUserType('type', Array)
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(User), 'type')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModel.php(159): CValidator->validate(Object(User), NULL)
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(794): CModel->validate(NULL)
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php(78): CActiveRecord->save()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#18 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://www.oxbridge.com/index.php?r=user/create
---
2013/11/13 12:30:40 [warning] [application] Failed to set unsafe attribute "password_repeat" of "User".
in /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php (77)
in /home/lunjiang/workspace/oxbridge/oxbridge/index.php (13)
2013/11/13 12:30:56 [warning] [application] Failed to set unsafe attribute "password_repeat" of "User".
in /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php (77)
in /home/lunjiang/workspace/oxbridge/oxbridge/index.php (13)
2013/11/13 12:30:56 [error] [exception.CException] exception 'CException' with message 'User and its behaviors do not have a method or closure named "setUserType".' in /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CComponent.php:265
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(225): CComponent->__call('setUserType', Array)
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): CActiveRecord->__call('setUserType', Array)
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CInlineValidator.php(42): User->setUserType('type', Array)
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/validators/CValidator.php(213): CInlineValidator->validateAttribute(Object(User), 'type')
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CModel.php(159): CValidator->validate(Object(User), NULL)
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(794): CModel->validate(NULL)
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php(78): CActiveRecord->save()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): UserController->actionCreate()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#10 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('user/create')
#18 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#19 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CApplication->run()
#20 {main}
REQUEST_URI=/index.php?r=user/create
HTTP_REFERER=http://www.oxbridge.com/index.php?r=user/create
---
2013/11/13 12:31:07 [warning] [application] Failed to set unsafe attribute "password_repeat" of "User".
in /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php (77)
in /home/lunjiang/workspace/oxbridge/oxbridge/index.php (13)
2013/11/13 17:52:32 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (/home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/main.php:51)
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(76): SiteController->renderFile()
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput()
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(206): CContentDecorator->run()
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(300): SiteController->endWidget()
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/column1.php(6): SiteController->endContent()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(126): require()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(784): SiteController->renderFile()
#10 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/SiteController.php(32): SiteController->render()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams()
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): SiteController->runAction()
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): SiteController->runActionWithFilters()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): SiteController->run()
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController()
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CWebApplication->run()
#19 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/cli/commands/ShellCommand.php(78): require()
#20 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleCommandRunner.php(67): ShellCommand->run()
#21 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleApplication.php(91): CConsoleCommandRunner->run()
#22 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CConsoleApplication->processRequest()
#23 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic.php(33): CConsoleApplication->run()
#24 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic(14): require_once()
REQUEST_URI=/index.php
2013/11/13 17:53:09 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (/home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/main.php:51)
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(76): SiteController->renderFile()
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput()
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(206): CContentDecorator->run()
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(300): SiteController->endWidget()
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/column1.php(6): SiteController->endContent()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(126): require()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(784): SiteController->renderFile()
#10 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/SiteController.php(32): SiteController->render()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams()
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): SiteController->runAction()
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): SiteController->runActionWithFilters()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): SiteController->run()
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController()
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CWebApplication->run()
#19 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/cli/commands/ShellCommand.php(78): require()
#20 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleCommandRunner.php(67): ShellCommand->run()
#21 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleApplication.php(91): CConsoleCommandRunner->run()
#22 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CConsoleApplication->processRequest()
#23 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic.php(33): CConsoleApplication->run()
#24 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic(14): require_once()
REQUEST_URI=/index.php
2013/11/13 12:53:34 [warning] [application] Failed to set unsafe attribute "password_repeat" of "User".
in /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/UserController.php (77)
in /home/lunjiang/workspace/oxbridge/oxbridge/index.php (13)
2013/11/13 17:55:47 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (/home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/main.php:51)
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(76): SiteController->renderFile()
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput()
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(206): CContentDecorator->run()
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(300): SiteController->endWidget()
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/column1.php(6): SiteController->endContent()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(126): require()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(784): SiteController->renderFile()
#10 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/SiteController.php(32): SiteController->render()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams()
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): SiteController->runAction()
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): SiteController->runActionWithFilters()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): SiteController->run()
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController()
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CWebApplication->run()
#19 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/cli/commands/ShellCommand.php(78): require()
#20 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleCommandRunner.php(67): ShellCommand->run()
#21 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleApplication.php(91): CConsoleCommandRunner->run()
#22 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CConsoleApplication->processRequest()
#23 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic.php(33): CConsoleApplication->run()
#24 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic(14): require_once()
REQUEST_URI=/index.php
2013/11/13 17:56:01 [error] [php] date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. (/home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/main.php:51)
Stack trace:
#0 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#1 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(76): SiteController->renderFile()
#2 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate()
#3 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput()
#4 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(206): CContentDecorator->run()
#5 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(300): SiteController->endWidget()
#6 /home/lunjiang/workspace/oxbridge/oxbridge/protected/views/layouts/column1.php(6): SiteController->endContent()
#7 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(126): require()
#8 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SiteController->renderInternal()
#9 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(784): SiteController->renderFile()
#10 /home/lunjiang/workspace/oxbridge/oxbridge/protected/controllers/SiteController.php(32): SiteController->render()
#11 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): SiteController->actionIndex()
#12 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams()
#13 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(286): SiteController->runAction()
#14 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CController.php(265): SiteController->runActionWithFilters()
#15 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): SiteController->run()
#16 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController()
#17 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#18 /home/lunjiang/workspace/oxbridge/oxbridge/index.php(13): CWebApplication->run()
#19 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/cli/commands/ShellCommand.php(78): require()
#20 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleCommandRunner.php(67): ShellCommand->run()
#21 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/console/CConsoleApplication.php(91): CConsoleCommandRunner->run()
#22 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CConsoleApplication->processRequest()
#23 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic.php(33): CConsoleApplication->run()
#24 /home/lunjiang/workspace/yii_base/yii-1.1.13.e9e4a0/framework/yiic(14): require_once()
REQUEST_URI=/index.php
